dashboard/test/ui/features/teacher_dashboard/teacher_dashboard.feature

```diff
@@ -40,8 +40,10 @@ Feature: Using the teacher dashboard
     And I click selector "#uitest-teacher-dashboard-nav a:contains(Manage Students)" once I see it
     And I wait until element "#uitest-manage-students-table" is visible
     And element "#uitest-manage-students-table tr:eq(1)" contains text "Sally"
+    And I wait until element "#uitest-privacy-text" is visible
+    And element "#uitest-privacy-text" contains text "Protecting your students' privacy"
```

Comment:
Looks like you introduced a reference to an element `#uitest-privacy-text` but we never gave that id to any element.

Reply:
Oh shoot, I forgot to commit that. Will do that now!